helm_deploy/hmpps-book-a-video-link-api/values.yaml

```diff
@@ -68,7 +68,6 @@ generic-service:
   allowlist:
     groups:
       - internal
-      - prisons
 
```

Comment:
Should cloud-platform IPs be on this list, so the UI can call in?  Or does the group `internal` cover that?